src/pages/LandingPage/TradingInfo.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useMemo, useState } from 'react';
+import { Typography, Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { Skeleton } from '@material-ui/lab';
+import { StakeQuickModal } from 'components';
+import { useLairInfo, useTotalRewardsDistributed } from 'state/stake/hooks';
+import { getDaysCurrentYear, formatCompact } from 'utils';
+
+const useStyles = makeStyles(({ palette, breakpoints }) => ({
+  tradingSection: {
+    background: palette.background.default,
+    width: 'calc(25% - 24px)',
+    maxWidth: 288,
+    minWidth: 220,
+    height: 133,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: 20,
+    margin: 12,
+    '& h6': {
+      paddingTop: 2,
+    },
+    '& p': {
+      fontSize: '12px',
+      color: palette.text.secondary,
+      paddingTop: '12px',
+    },
+    '& h4': {
+      fontSize: 12,
+      cursor: 'pointer',
+      marginTop: 3,
+      color: palette.primary.main,
+    },
+    [breakpoints.down('xs')]: {
+      height: 'unset',
+    },
+  },
+}));
+
+export const TradingInfo: React.FC<{ globalData: any }> = ({ globalData }) => {
+  const classes = useStyles();
+  const lairInfo = useLairInfo();
+  const daysCurrentYear = getDaysCurrentYear();
+  const [openStakeModal, setOpenStakeModal] = useState(false);
+
+  const APR =
+    (((Number(lairInfo?.oneDayVol) * 0.04 * 0.01) /
+      Number(lairInfo?.dQuickTotalSupply.toSignificant(6))) *
+      daysCurrentYear) /
+    (Number(lairInfo?.dQUICKtoQUICK.toSignificant()) *
+      Number(lairInfo?.quickPrice));
```

Comment:
Don't we have a function to get the USD price instead of converting the dQuick price manually?

Reply:
update the function to get dQUICK APY